src/pages/community/download.md

```diff
@@ -29,14 +29,15 @@ The downloads for the latest release are distributed via mirror sites.
-| v0.8.0  | 2020-05-04 | [Download](https://archive.apache.org/dist/incubator/yunikorn/0.8.0-incubating/apache-yunikorn-0.8.0-incubating-src.tar.gz) <br />[Checksum](https://archive.apache.org/dist/incubator/yunikorn/0.8.0-incubating/apache-yunikorn-0.8.0-incubating-src.tar.gz.sha512) & [Signature](https://archive.apache.org/dist/incubator/yunikorn/0.8.0-incubating/apache-yunikorn-0.8.0-incubating-src.tar.gz.asc) | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-0.8.0/images/sha256-0b35f9bb767f06af7f84f58799401ba7de7b8991f3c9724f40f733bc517193df) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-0.8.0/images/sha256-700e9bf7bc5597ab144be9f29b489fb82d7e012ee46d34bbc26cfb91bf364124) <br />[web](https://hub.docker.com/layers/apache/yunikorn/web-0.8.0/images/sha256-83faa83ec9d1c90b40ca5bee9977c31fba31ba34f3ae9c785d994adbb545a273) | [Announcement](/release-announce/0.8.0) |
+The latest release of Apache YuniKorn is v0.12.1.
+
+| Version | Release date | Source download                                                                                                                                                                                                                                                                                                                                                                                         | Docker images  | Release notes  |
+|---------|--------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ----  | ----  |
+| v0.12.1 | 2021-12-17   | [Download](https://www.apache.org/dyn/closer.lua/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz) <br />[Checksum](https://downloads.apache.org/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz.asc)                            | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-0.11.0/images/sha256-7d156e4df2cb1a99d6f3cf5bfd15ae42c7c195f66411b83a720b375194209d20) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-0.11.0/images/sha256-2b1cee3e79a0f08c835ed264c537b14eb0527d7196dcbbf613296f034c8c2a70 )<br />[web](https://hub.docker.com/layers/apache/yunikorn/web-0.11.0/images/sha256-e07a8465fefb4f51ab989b7be4db824b51fc4b925fb400c09fad87d0b0729246) | [Announcement](/release-announce/0.11.0) |
```

Comment:
* all image links point to 0.11.0 should be 0.12.1
* announcement points to 0.11.0
* The date for release should be updated to the date the IPMC approves the release, that is when we officially release the code